src/container/srv_container.c

```diff
@@ -14,17 +14,18 @@
 
 #define D_LOGFAC DD_FAC(container)
 
-#include <daos_srv/container.h>
```

Comment:
[Request changes] This is the module API header of `ds_cont`, and was included first so that if this header forgot to include some required headers, the compilation of `ds_cont` would fail.

Reply:
We spoke about this in some detail, I had misunderstood some of the description inhttps://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes which I was trying to follow.  I've taken the PR back to draft and will work through the issues again.